app/models/article.rb

```diff
@@ -16,4 +18,8 @@ class Article < ApplicationRecord
   def update_published_at
     self.published_at = self.is_draft ? nil : DateTime.now
   end
+
+  def author_name
+    "#{author.first_name} #{author.last_name}"
```

Comment:
Yeah it's a thing of graphql. I can run a method on the object we're spitting out, but im not so sure i can run it on the object related to it. I'll have this method call out to the user, though. which is a bit fiddly because it's two levels of indirection but is otherwise ok :)